src/main/java/net/mcreator/ui/MCreatorApplication.java

```diff
@@ -156,6 +157,8 @@ private MCreatorApplication(List<String> launchArguments) {
 		splashScreen.setProgress(60, "Preloading resources");
 		TiledImageCache.loadAndTileImages();
 
+		MCREvent.event(new PreGeneratorsLoadingEvent(this));

```

Comment:
Put this below 

`splashScreen.setProgress(70, "Loading generators");`

or plugin crashing will make splash stuck on

Preloading resources

which is reserved for, well, resources, not generators

Reply:
It is ok before generators, just text is misleading for the stated reason.

Plugin throws exception while adding eg. new mod element, and user sees:

Preloading resources

stuck on splash screen. Is this ok?

Reply:
👍